ansible/roles/php/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Update apt
+  become: yes
```

Comment:
Look at [the full file](https://github.com/NikitaStrelkov/DesignPatternsPHP/blob/a7f40a2083e6790c3f3f4dcbc3fefe239a9ab338/ansible/roles/php/tasks/main.yml). The 'become' and 'sudo' used together. And, in the same time, php-task already [called with sudo](https://github.com/NikitaStrelkov/DesignPatternsPHP/blob/a7f40a2083e6790c3f3f4dcbc3fefe239a9ab338/ansible/playbook.yml) in the playbook.yml.

I think, all "become" and "sudo" attributes should be removed from tasks and used in playbook.